src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java

```diff
@@ -35,11 +35,21 @@ public void testEmptyBlocksAndCatchBlocks() throws Exception {
         verifyWithWholeConfig(getPath("InputEmptyBlocksAndCatchBlocks.java"));
     }
 
+    @Test
+    public void testEmptyBlocksAndCatchBlocksFormatted() throws Exception {
+        verifyWithWholeConfig(getPath("InputFormattedEmptyBlocksAndCatchBlocks.java"));
+    }
+
     @Test
     public void testEmptyFinallyBlocks() throws Exception {
         verifyWithWholeConfig(getPath("InputEmptyFinallyBlocks.java"));
     }
 
+    @Test
+    public void testEmptyFinallyBlocksFormatted() throws Exception {
+        verifyWithWholeConfig(getPath("InputFormattedEmptyFinallyBlocks.java"));
```

Comment:
this file is identical to original Input, please remove it from exclude.

Reply:
done